### ISSUE TITLE
Home Manager module for ActivityWatch

### ISSUE THREAD

**Issue by jackdbd** [jackdbd/nix-config#3]
Create an Home Manager module that allows to configure [ActivityWatch](https://docs.activitywatch.net/en/latest/) using Nix expressions instead of TOML files.

Where to store ActivityWatch [categorization rules](https://docs.activitywatch.net/en/latest/features/categorization.html)? I can export/import them as a JSON, but how to do it automatically.

**Comment by jackdbd** [comment]
Implemented first version in fd81e50b20e6332bb3794446132396d1266a089c. At the moment all ActivityWatch components (aw-qt, aw server, aw watchers) can be configured only via TOML files.

**Comment by jackdbd** [comment]
Implemented in #7 .

**Comment by rbozan** [comment]
Hey @jackdbd , I was wondering if it was possible to get this in Home Manager somehow, as I'd also like to use this as a systemd service.

**Comment by jackdbd** [comment]
I initially thought about opening a PR in the Home Manager repo, but then I discussed about my solution in the [NixOS forum](https://discourse.nixos.org/t/systemd-service-for-activitywatch/37649
) and in the [ActivityWatch forum](https://forum.activitywatch.net/t/nix-home-manager-module-for-activitywatch/2753) and found out that ActivityWatch is meant to be started with the autostart mechanism of the user's desktop environment.

The rationale behind the decision of not including a systemd service in ActivityWatch is explained in [this comment by one of ActivityWatch's authors](https://github.com/ActivityWatch/activitywatch/issues/403#issuecomment-609463584).

So, if you want to use the systemd service I implemented in #7, I think your best bet is just copy-pasting my Home Manager module in your Nix configuration.
